supertokens_python/utils.py

```diff
@@ -195,28 +192,6 @@ def find_first_occurrence_in_list(
     return None
 
 
-def execute_async(mode: str, func: Callable[[], Coroutine[Any, Any, None]]):
-    real_mode = None
-    try:
-        asyncio.get_running_loop()
-        real_mode = "asgi"
-    except RuntimeError:
-        real_mode = "wsgi"
-
-    if mode != real_mode:
-        warnings.warn(
-            "Inconsistent mode detected, check if you are using the right asgi / wsgi mode",
-            category=RuntimeWarning,
-        )
-
-    if real_mode == "wsgi":
-        asyncio.run(func())
-    else:
-        check_event_loop()
-        loop = asyncio.get_event_loop()
-        loop.create_task(func())
```

Comment:
This function wasn't used anywhere throughout the code. We discussed removing this last time.